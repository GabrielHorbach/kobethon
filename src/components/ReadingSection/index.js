import React from 'react';

import BookAnimation from '../../assets/animations/book_animation.gif';

import Container from './styles';

function ReadingSection() {
  return (
    <Container>
      <div>
        <p>Meu outro hobbie favorito Ã© <strong>ler</strong>.</p><br />
        <p>Amo livros de fantasia, com destaque aqui para Harry Potter e Percy Jackson que eu nunca canso de reler.</p>
      </div>
      <img src={BookAnimation} alt="Book pages gif" />
    </Container>
  );
}

export default ReadingSection;
