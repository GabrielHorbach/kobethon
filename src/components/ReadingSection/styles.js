import styled from 'styled-components';

const Container = styled.section`
  width: 100%;
  height: 100vh;
  background: #67D493;
  display: flex;
  align-items: center;
  padding: 0 15rem;

  @media (max-width: 1200px) {
    padding: 0 3rem 0 5rem;
  }

  @media (max-width: 650px) {
    flex-direction: column;
    padding: 0 3rem 10%;
    height: auto;
  }

  div {
    width: 50%;

    @media (max-width: 650px) {
      width: 100%;
      margin-top: 7%;
    }

    p {
      font-size: 1.5rem;
      line-height: 2.5rem;
      color: #f1f1f1;

      @media (max-width: 1100px) {
        font-size: 1.3rem;
        line-height: 2.3rem;
      }

      @media (max-width: 800px) {
        font-size: 1.2rem;
        line-height: 2.2rem;
      }
    }
  }

  img {
    width: 50%;
    margin-top: -2.5rem;

    @media (max-width: 650px) {
      margin-top: 2rem;
      width: 60%;
    }
  }
`;

export default Container;