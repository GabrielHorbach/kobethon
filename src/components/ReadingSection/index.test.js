import React from 'react';

import { shallow } from 'enzyme';

import ReadingSection from './index';

describe('ReadingSection Section', () => {
  const wrapper = shallow(<ReadingSection />);

  it('should render without errors', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render gif', () => {
    const gif = wrapper.find('img');
    expect(gif).toBeTruthy();
  });
});