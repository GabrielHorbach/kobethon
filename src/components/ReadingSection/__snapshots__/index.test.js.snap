// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadingSection Section should render without errors 1`] = `ShallowWrapper {}`;
